src/test/java/de/tum/in/www1/artemis/exam/StudentExamIntegrationTest.java

```diff
@@ -409,7 +409,9 @@ void testGetStudentExamForConduction() throws Exception {
             assertThat(response).isEqualTo(studentExam);
             assertThat(response.isStarted()).isTrue();
             assertThat(response.getExercises()).hasSize(exam2.getNumberOfExercisesInExam());
-
+            for (Exercise exercise : response.getExercises()) {
+                assertThat(exercise.getExerciseGroup()).isNotNull();
```

Comment:
please verify that course in exerciseGroup is null

Reply:
do you mean the course in exerciseGroup.exam because an exercise group doesn't have a course as direct attribute.